R/lazy.R

```diff
@@ -118,7 +117,7 @@ print.tbl_connect <- function(x, ..., n = NULL) {
 
 #' @export
 as.data.frame.tbl_connect <- function(x, row.names = NULL, optional = NULL, ..., n = Inf) {
-  as.data.frame(collect(x, n = n))
+  as.data.frame(dplyr::collect(x, n = n))
```

Comment:
```suggestion
  # We don't need to check if dplyr is available here
  # because you won't have a tbl_connect without first
  # checking for dplyr.
  as.data.frame(dplyr::collect(x, n = n))
```